ghcide/src/Development/IDE/Main.hs

```diff
@@ -11,127 +12,146 @@ module Development.IDE.Main
 ,testing
 ,Log(..)
 ) where
-import           Control.Concurrent.Extra              (withNumCapabilities)
```

Comment:
This diff is way too big, I'm going to fix this

Reply:
fixed it

Reply:
This has been resolved: on the master branch there is a commit that ran stylish-haskell on that module